src/components/views/PCI.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  Headline,
+  Layout,
+  Pane,
+  PaneMenu,
+  Row,
+  Spinner,
+} from '@folio/stripes/components';
+
+import { IfPermission } from '@folio/stripes/core';
+
+import {
+  Agreements,
+  AgreementsList,
+  PCICoverage,
+  PCIInfo,
+  ParentPackageDetails,
+  TitleInfo,
+} from '../EResourceSections';
+
+export default class PCI extends React.Component {
+  static propTypes = {
+    data: PropTypes.shape({
+      eresource: PropTypes.shape({
+        id: PropTypes.string,
+        name: PropTypes.string,
+        status: PropTypes.shape({
+          label: PropTypes.string,
+        }),
+      }).isRequired,
+      relatedEntitlements: PropTypes.array,
+    }),
+    handlers: PropTypes.shape({
+      onClose: PropTypes.func.isRequired,
+    }),
+    isLoading: PropTypes.bool,
+  }
+
+  state = {
+    sections: {
+      coverage: true,
+      pciagreements: true
+    },
+  }
+
+  getSectionProps = (id) => {
+    const { data } = this.props;
+
+    return {
+      data,
+      id,
+      pci: data.eresource,
+      onToggle: this.handleSectionToggle,
+      open: this.state.sections[id],
+    };
+  }
+
+  handleSectionToggle = ({ id }) => {
+    this.setState((prevState) => ({
+      sections: {
+        ...prevState.sections,
+        [id]: !prevState.sections[id],
+      }
+    }));
+  }
+
+  handleAllSectionsToggle = (sections) => {
+    this.setState({ sections });
+  }
+
+  renderLoadingPane = () => {
+    return (
+      <Pane
+        defaultWidth="45%"
+        dismissible
+        id="pane-view-pci"
+        onClose={this.props.handlers.onClose}
+        paneTitle={<FormattedMessage id="ui-agreements.loading" />}
+      >
+        <Layout className="marginTop1">
```

Comment:
Use the stripes-components [`LoadingPane` instead.](https://github.com/folio-org/ui-erm/blob/1d76c8608c31bc74729d7e287956e614511344f1/src/components/views/Agreement.js#L243)